### PR TITLE
December 22 release notes

### PR DESCRIPTION
**Pull request by olenagerasimova**
Please, check the December blog post. But please do not merge, release is not ready yet.

### CODE REVIEW

File: _posts/2022/12/2022-12-22-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - Micrometer metrics
+  - Prometheus
+  - JFR
+---
+
+At the end of November, [Dzone](https://dzone.com/) has published our article:
+[Private Remote Maven Repository With Artipie](https://dzone.com/articles/private-remote-maven-repository-with-artipie-1). 
+We will be happy to get any feedback!
+
+Artipie new version [v0.28.0](https://github.com/artipie/artipie/releases/tag/v0.28.0) was released on December 28.
```
Comment (dgarus) [inline _posts/2022/12/2022-12-22-release-notes.md]: `December 28` -> `December 20` ?

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
One minor comment

**Comment by olenagerasimova** [comment]
@dgarus today's 21, so I've corrected to 21))

**Review by dgarus — APPROVED** [review]
Great! Thanks

**Pull request merged**